ipaserver/install/httpinstance.py

```diff
@@ -593,6 +593,7 @@ def start_tracking_certificates(self):
                 post_command='restart_httpd', storage='FILE',
                 profile=dogtag.DEFAULT_PROFILE,
                 pinfile=key_passwd_file,
+                dns=[self.fqdn, f'{IPA_CA_RECORD}.{api.env.domain}'],
```

Comment:
The upgrader code uses ``f'{ipautil.format_netloc(api.env.domain)}'`` instead of ``f'{api.env.domain}'``.

Reply:
@tiran that is a good pickup. AFAICT it should not make a difference, because (in theory) it must be a hostname and not an IPv6 address.

I looked at the git blame and can't perceive a clear reason why it was written this way in `upgrade.py`, except perhaps an abundance of caution in case it was somehow an IPv6 address.

Do you perfer to remove the application of `ipautil.format_netloc` and just use plain `f{api.env.doman}`.  Or add `ipautil.format_netloc(...)` to the other places?  Or just ignore the difference?